packages/docs-reanimated/static/examples/FloatingActionButton.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+import { StyleSheet, SafeAreaView, View, Pressable, Text } from 'react-native';
+import Animated, {
+  withDelay,
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+  withTiming,
+} from 'react-native-reanimated';
+
+const AnimatedPressable = Animated.createAnimatedComponent(Pressable);
+
+const SPRING_CONFIG = {
+  duration: 1200,
+  overshootClamping: true,
+  dampingRatio: 0.8,
+};
+
+const OFFSET = 60;
+
+const FloatingActionButton = ({ isExpanded, index, buttonLetter }) => {
+  const animatedStyles = useAnimatedStyle(() => {
+    const moveValue = interpolate(
+      Number(isExpanded.value),
+      [0, 1],
+      //highlight-next-line
+      [0, OFFSET * index]
+    );
```

Comment:
You can simply replace it with this:
```suggestion
const moveValue = isExpanded.value ? OFFSET * index : 0;
```

https://github.com/software-mansion/react-native-reanimated/assets/36106620/fcd7e4d1-741e-4ae8-b517-927f3afb66f2